pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+<!-- 
+- make sure you follow our [coding style guidelines][https://github.com/bisq-network/style/issues)
+- provide some meaningful info below
+- create the PR
+- in case you receive a "Change request" and/or a NACK, please react within 30 days. If not, we will close your PR and it can not be up for compensation.
+- After addressing the change request, __please re-request a review!__ Otherwise we might miss your PR as we tend to only look at the PR where there is a review required.
+-->
+
+Fixes #, fixes #
```

Comment:
Since this is targetted at new committers. It might be good to explain a bit about how GitHub uses the `fixes` keyword to onboard new users.

Maybe something similar to: `GitHub can automatically close or tag issues that are listed in the Pull Request. To automatically close a bug use fixes #<issue-number>. To reference an issue, but not close it, use: #<issue-number>`